monitoring/tracer/uss_receiver/routes.py

```diff
@@ -50,7 +50,7 @@ def rid_isa_notification(id: str) -> Tuple[str, int]:
   label = colored('ISA', 'cyan')
   try:
     json = flask.request.json
-    if 'service_area' in json:
+    if 'service_area' in json and json['service_area']:
```

Comment:
if json.get('service_area') ?

Reply:
Ah yes...I had been thinking `get` behaved differently.  Changed here and a few other places -- thanks!